evalml/tests/automl_tests/test_default_algorithm.py

```diff
@@ -381,3 +383,193 @@ def test_select_cat_cols(
             assert value["columns"] == algo._selected_cols
         elif "Categorical Pipeline - Select Columns Transformer" in component:
             assert value["columns"] == algo._selected_cat_cols
+
+
+@pytest.mark.parametrize(
+    "automl_type",
+    [
+        ProblemTypes.TIME_SERIES_BINARY,
+        ProblemTypes.TIME_SERIES_MULTICLASS,
+        ProblemTypes.TIME_SERIES_REGRESSION,
+    ],
+)
+@patch("evalml.pipelines.components.FeatureSelector.get_names")
+def test_default_algorithm_time_series(
+    mock_get_names, automl_type, ts_data, ts_data_binary, ts_data_multi
+):
+    if automl_type == ProblemTypes.TIME_SERIES_BINARY:
+        X, y = ts_data_binary
+    elif automl_type == ProblemTypes.TIME_SERIES_MULTICLASS:
+        X, y = ts_data_multi
+    elif automl_type == ProblemTypes.TIME_SERIES_REGRESSION:
+        X, y = ts_data
+
+    mock_get_names.return_value = ["0", "1", "2"]
+    problem_type = ProblemTypes.TIME_SERIES_REGRESSION
+    sampler_name = None
+    pipeline_params = {
+        "pipeline": {
+            "time_index": "date",
+            "gap": 1,
+            "max_delay": 3,
+            "delay_features": False,
+            "forecast_horizon": 10,
+        }
+    }
+
+    algo = DefaultAlgorithm(
+        X, y, problem_type, sampler_name, pipeline_params=pipeline_params
+    )
+    naive_model_families = set([ModelFamily.LINEAR_MODEL, ModelFamily.RANDOM_FOREST])
+
+    first_batch = algo.next_batch()
+    assert len(first_batch) == 2
+    assert {p.model_family for p in first_batch} == naive_model_families
+    for pipeline in first_batch:
+        assert pipeline.parameters["pipeline"] == pipeline_params["pipeline"]
+        assert pipeline.parameters["DateTime Featurization Component"]["time_index"]
```

Comment:
As a heads up, since [3192](https://github.com/alteryx/evalml/pull/3192) is in, this component is now called `DateTime Featurizer`.

Reply:
ah thank you, will update!